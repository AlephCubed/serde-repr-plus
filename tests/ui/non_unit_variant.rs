use serde_repr_plus::Serialize_repr_clamp;

#[derive(Serialize_repr_clamp)]
#[repr(u8)]
enum SmallPrime {
    Two(u8),
    Three(u8),
    Five(u8),
    Seven(u8),
}

fn main() {}
