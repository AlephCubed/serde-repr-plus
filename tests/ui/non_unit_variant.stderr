error: must be a unit variant to use serde_repr_plus derive
 --> tests/ui/non_unit_variant.rs:6:5
  |
6 |     Two(u8),
  |     ^^^
